knapsack/tests/test_greedy_knapsack.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from . import greedy_knapsack as kp
+from knapsack import greedy_knapsack as kp
```

Comment:
Is this a vital change?  There are two ways to go.  Hardcode the name of the parent directory or use relative imports.  Each has its pros and cons.  Relative imports are better for easy refactoring and absolute imports are better for libraries and projects that need stability.

Reply:
Relative import would be like so: `from .. import greedy_knapsack as kp` as I created a `tests` directory for both test files. Here's the tree:
```
knapsack/
├── README.md
├── __init__.py
├── greedy_knapsack.py
├── knapsack.py
└── tests
    ├── __init__.py
    ├── test_greedy_knapsack.py
    └── test_knapsack.py
```

Reply:
I suppose for our use case relative imports should be de facto as we don't know from where the script is going to be called by the user.